flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -40,15 +43,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Map;
 import java.util.ArrayDeque;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.LinkedHashMap;
 import java.util.HashMap;
 import java.util.Iterator;
-import java.util.LinkedHashMap;
```

Comment:
Let's keep these sorted. I use and recommend @StephanEwen's configuration from https://github.com/apache/flink/pull/2352/files